evolver/calibration/interface.py

```diff
@@ -125,6 +125,9 @@ class state(_BaseConfig):
     class Config(Transformer.Config):
         input_transformer: ConfigDescriptor | Transformer | None = None
         output_transformer: ConfigDescriptor | Transformer | None = None
+        calibration_file: str | None = None
```

Comment:
@amitschang I've opened # to address this but we had discussed this a few times before and decided that we didn't want to add this as it's not necessary and self-referential.

All configs can be loaded from file using ``model_validate``, the calibration files are no exception here.
See https://github.com/ssec-jhu/evolver-ng/blob/main/evolver/base.py#L52-L53
```python
class _BaseConfig(pydantic.BaseModel):
    model_config = pydantic.ConfigDict(extra="ignore", from_attributes=True)

    @classmethod
    def model_validate(cls, obj, *, strict=None, from_attributes=None, context=None):
        """Effectively the same as pydantic.BaseModel.model_validate() except that it automatically handles json, and
        conversion from instances of ``BaseConfig`` and ``BaseInterface`
        """
        if obj is None:
            return cls()
        elif isinstance(obj, os.PathLike):
            return cls.load(file_path=obj)
        ...
```

It's quite possible that there are a few code paths that need patching or re-directing to ensure that this works from all nominal invocations, # can contain that if needed.